respecConfig.js

```diff
@@ -16,9 +15,7 @@ var respecConfig = {
     height: 100,
     id: 'spec-logo',
   }],
-  wg: "Web Platform Incubator Community Group",
```

Comment:
This removal is causing ReSpec errors for me locally: "wg configuration option is required for this kind of document."

Reply:
No longer having this problem today, presumably because the xref database is up.